Reference/using-ioc.md

```diff
@@ -4,7 +4,7 @@ _This section will show you how to setup Ioc/Dependency Injection with your Umbr
 
 ##Overview
 
-We don't use IoC in the Umbraco source code. This isn't because we don't like it or don't want to use it, it's because we want you as a developer to be able to use whatever IoC framework that you would like to use without jumping through any hoops. With that said, it means it is possible to implement whatever IoC engine that you'd like!
+Since Umbraco 7 we use IoC in the Umbraco source code for the backend site. With that said, it is possible to implement whatever IoC engine that you'd like you just need to re-register our api controllers!
```

Comment:
No, Umbraco does not use IoC whatsoever in it's codebase. 

WebApi and MVC do not require IoC to execute, this is an optional thing for developers if they want that functionality. There's not 're-registering' of api controllers... that is simply required if you choose to use IoC. We don't 'register' controllers anywhere, this is simply handled by WebApi/MVC controller factories. When you implement IoC, these controller factories delegate to your IoC container.

So, if you can update these docs by reverting the description change and also changing your text:

> //re-register umbraco webapi controllers used by the admin site

to simply 

> //register umbraco webapi controllers used by the admin site

We'll pull these docs in asap.